server/src/main/java/org/candlepin/resource/JobResource.java

```diff
@@ -145,20 +144,15 @@ else if (!StringUtils.isEmpty(principalName)) {
         }
 
         return this.translator.translateQuery(
-            statuses.transform(new ElementTransformer<JobStatus, JobStatus>() {
-                @Override
-                public JobStatus transform(JobStatus jobStatus) {
-                    return jobStatus.cloakResultData(true);
-                }
-            }), JobStatusDTO.class);
+            statuses.transform(jobStatus -> jobStatus.cloakResultData(true)), JobStatusDTO.class);
```

Comment:
:+1:

I didn't realize the compiler was smart enough to do all of this with a single operator. Good to know.

Reply:
Yep, its because ElementTransformer happens to be a functional interface (has exactly 1 abstract method, so it knows whatever comes after -> is basically the implementation of this one method).